src/foam/u2/table/UnstyledTableGroup.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.table',
+  name: 'UnstyledTableGroup',
+  extends: 'foam.u2.View',
+  mixins: ['foam.u2.table.TableHelperMixin'],
+
+  requires: [
+    'foam.u2.CheckBox',
+    'foam.core.SimpleSlot',
+    'foam.u2.view.OverlayActionListView'
+  ],
+
+  imports: [
+    'props',
+    'propertyNamesToQuery',
+    'canBuildObjfromProj',
+    'nestedPropertyNamesAndItsIndexes'
+  ],
+
+  properties: [
+    'obj',
+    'projection',
+  ],
+
+  methods: [
+    function render() {
+      var self = this;
+      var objForCurrentProperty = this.obj;
+      var expr = foam.mlang.Expressions.create();
+      var nestedPropertyValues = this.columnHandler.filterOutValuesForNotNestedProperties(this.projection, this.nestedPropertyNamesAndItsIndexes[1]);
+      var nestedPropertiesObjsMap = this.columnHandler.groupObjectsThatAreRelatedToNestedProperties(this.data.of, this.nestedPropertyNamesAndItsIndexes[0], nestedPropertyValues);
```

Comment:
Could you pick shorter names, here and elsewhere? Like groupRelatedProperties() maybe?